<!-- MHonArc v2.6.19+ -->
<!--X-Subject: Re: Issue in ModelFree Calculation -->
<!--X-From-R13: &#60;nevgenorwNpfvevvpo.va> -->
<!--X-Date: Fri, 26 Aug 2016 18:26:15 +0200 -->
<!--X-Message-Id: 1686962642.387128.1472228742399@mail.yahoo.com -->
<!--X-Content-Type: text/plain -->
<!--X-Reference: 419270419.2002127.1469020706363.JavaMail.yahoo.ref@mail.yahoo.com -->
<!--X-Reference: 419270419.2002127.1469020706363.JavaMail.yahoo@mail.yahoo.com -->
<!--X-Reference: CA+CBx2NOPpSuwaybrdkGnuoYEoY9TX4eoaLOMra94N0P&#45;AFRSA@mail.gmail.com -->
<!--X-Reference: CAED9pY&#45;3zbQoTj+_xy&#45;AEcv3+bHBX9sEve5nQDpD86&#45;E0AReUQ@mail.gmail.com -->
<!--X-Head-End-->
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
   "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>Re: Issue in ModelFree Calculation -- August 26, 2016 - 18:26</title>
<link rel="stylesheet" type="text/css" href="/mail.gna.org/archives-color-gna.css"> 
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
</head>
<body>
<!--X-Body-Begin-->
<!--X-User-Header-->
<!--X-User-Header-End-->
<!--X-TopPNI-->
<h2><img src="/mail.gna.org/images/mail.orig.png" width="48" height="48"
alt="mail" class="pageicon" />Re: Issue in ModelFree Calculation</h2>
<br />
<div class="topmenu">
<a href="../" class="tabs">Others Months</a> | <a href="index.html#00012" class="tabs">Index by Date</a> | <a href="threads.html#00012" class="tabs">Thread Index</a><br />
<span class="smaller">&gt;&gt;&nbsp;&nbsp;
[<a href="msg00011.html">Date Prev</a>] [<a href="msg00013.html">Date Next</a>] [<a href="msg00004.html">Thread Prev</a>] [<a href="msg00005.html">Thread Next</a>]
</div>

<!--X-TopPNI-End-->
<!--X-MsgBody-->
<!--X-Subject-Header-Begin-->
<h3><a name="header" href="#header">Header</a></h3>
<!--X-Subject-Header-End-->
<!--X-Head-of-Message-->
<ul class="headdata">
<li class="menuitem">
<em>To</em>: Edward d'Auvergne &lt;edward@xxxxxxxxxxxxx&gt;,  &quot;relax-users@xxxxxxx&quot; &lt;relax-users@xxxxxxx&gt;</li>
<li class="menuitem">
<em>Date</em>: Fri, 26 Aug 2016 16:25:42 +0000 (UTC)</li>
<li class="menuitem">
<em>Dkim-signature</em>: v=1; a=rsa-sha256; c=relaxed/relaxed; d=yahoo.com; s=s2048; t=1472228743; bh=tJb7a+qyk5zwF9USLmdH0vQvNG9cOcEtSJi/Hv4j6eM=; h=Date:From:Reply-To:To:In-Reply-To:References:Subject:From:Subject; b=brzhK3GgpwB2p56UQbuikq0k3ZtLDer9c+VRqlp0gJgeY+0aU7jQkcCKSaML9bWwBUUg4prc6ez01SYp03QqxuRJDP/9RedMHUrNqrRsv2lKo8UmtlXgIQbIKXeHhlLYV+HFGWZD6N4ox/vXTFWGYd5secawKoaoMhqmQgYIAesoK5OMbpShm/dSwdNKJzlIU7xXSuBg/KWsr4Z0wsR+IQN2sXiSUSMQ3KiwhR1B6zWfmMDKjTXg8qKR2jJjPnu4/xjrqnrPy8eJ1yjgo8sBUzH92F7Pwdywy5kRq9/xBPg6UYrpv8sjiqAfgyYcLKWowL7qizZ+fGJ4IQx4R8BdwQ==</li>
<li class="menuitem">
<em>Message-id</em>: &lt;<a href="msg00012.html">1686962642.387128.1472228742399@mail.yahoo.com</a>&gt;</li>
<li class="menuitem">
<em>References</em>: &lt;419270419.2002127.1469020706363.JavaMail.yahoo.ref@mail.yahoo.com&gt; &lt;419270419.2002127.1469020706363.JavaMail.yahoo@mail.yahoo.com&gt; &lt;CA+CBx2NOPpSuwaybrdkGnuoYEoY9TX4eoaLOMra94N0P-AFRSA@mail.gmail.com&gt; &lt;<a href="msg00004.html">CAED9pY-3zbQoTj+_xy-AEcv3+bHBX9sEve5nQDpD86-E0AReUQ@mail.gmail.com</a>&gt;</li>
</ul>
<!--X-Head-of-Message-End-->
<!--X-Head-Body-Sep-Begin-->
</div><!-- end headdata -->
<br />
<h3><a name="content" href="#content">Content</a></h3>
<div class="postedby">Posted by <strong>aritrabej</strong> on August 26, 2016 - 18:26:</div>
<div class="msgdata">
<!--X-Head-Body-Sep-End-->
<!--X-Body-of-Message-->
<pre style="margin: 0em;">Hi Edward,Thank you for your suggestions. Extremely sorry for my late reply. 
I already tried what Troel had told me. As I didn't install the SciPy package 
(not required for ModelFree analysis), I am unable to perform the full test 
suite. As per your suggestion, I went through the wxPython demo, and found a 
missing link of a library package which I have successfully rebuild. Now 
wxPython is setup correctly. But still I get the same error like previous:
terminate called after throwing an instance of 'std::bad_alloc'  what():  
std::bad_allocAbort (core dumped)
This time I didn't get errors with the libraries. I have also attached the 
output of &quot;$relax -i&quot;. Kindly suggest something to resolve this problem.
RegardsAritra

      From: Edward d'Auvergne &lt;edward@xxxxxxxxxxxxx&gt;
 To: Troels Emtekær Linnet &lt;tlinnet@xxxxxxxxxxxxx&gt; 
Cc: aritrabej@xxxxxxxxxxx; &quot;relax-users@xxxxxxx&quot; &lt;relax-users@xxxxxxx&gt;
 Sent: Monday, August 8, 2016 8:25 PM
 Subject: Re: Issue in ModelFree Calculation
  
On 20 July 2016 at 16:02, Troels Emtekær Linnet &lt;tlinnet@xxxxxxxxxxxxx&gt; wrote:
</pre><blockquote class="blockquote"><pre style="margin: 0em;">Hi.

Did you compile relax yourself?
Try compiled version of relax.

Have you tried running the systemtests?
<a  rel="nofollow" href="http://wiki.nmr-relax.com/Installation_test">http://wiki.nmr-relax.com/Installation_test</a>

It looks like you are running mpirun.
Try running without.

Or look here:
<a  rel="nofollow" href="http://wiki.nmr-relax.com/Run_relax_at_Google_Cloud_Computing">http://wiki.nmr-relax.com/Run_relax_at_Google_Cloud_Computing</a>

<a  rel="nofollow" href="http://wiki.nmr-relax.com/OpenMPI#Relax_In_multiprocessor_mode">http://wiki.nmr-relax.com/OpenMPI#Relax_In_multiprocessor_mode</a>

2016-07-20 15:18 GMT+02:00 &lt;aritrabej@xxxxxxxxxxx&gt;:

</pre><blockquote class="blockquote"><pre style="margin: 0em;">Dear all,I am a new user of Relax and I am trying to perform ModelFree
calculation with a 207 residue protein using both fully automated and
manual protocol of Relax GUI mode. But every time the calculation is
stopped with a &quot;bad_alloc&quot; error. The details of the is as follows:
terminate called after throwing an instance of 'std::bad_alloc'what():
  std::bad_alloc*** Process received signal ***Signal: Aborted (6)Signal
code:
  
(-6)[0x2e040c]/lib/libc.so.6(abort+0x17a)[0xb415ea]/usr/lib/libstdc++.so.6(_ZN9__gnu_cxx27__verbose_terminate_handlerEv+0x167)[0x65c0397]/usr/lib/libstdc++.so.6(+0x57f226)[0x65be226]/usr/lib/libstdc++.so.6(+0x57f263)[0x65be263]/usr/lib/libstdc++.so.6(+0x57f3a2)[0x65be3a2]/usr/lib/libstdc++.so.6(_Znwj+0x87)[0x65be937]/usr/lib/libstdc++.so.6(_ZNSbIwSt11char_traitsIwESaIwEE4_Rep9_S_createEjjRKS1_+0x79)[0x65b35a9]/usr/lib/libstdc++.so.6(_ZNSbIwSt11char_traitsIwESaIwEE9_M_mutateEjjj+0x51)[0x65b5471]/usr/lib/libstdc++.so.6(_ZNSbIwSt11char_traitsIwESaIwEE15_M_replace_safeEjjPKwj+0x3b)[0x65b562b]/usr/lib/libstdc++.so.6(_ZNSbIwSt11char_traitsIwESaIwEE6assignEPKwj+0x5f)[0x65b56ef]/usr/local/lib/libwx_gtk2u_stc-3.0.so.0(+0x3a535)[0x1fc8535]/usr/local/lib/libwx_gtk2u_stc-3.0.so.0(+0x40714)[0x1fce714]/usr/local/lib/libwx_gtk2u_stc-3.0.so.0(+0x7c1bd)[0x200a1bd]/usr/local/lib/libwx_gtk2u_stc-3.0.so.0(+0x7c38f)[0x200a38f]/usr/local/lib/libwx_gtk2u_stc-3.0.so.0(+0x83a3f)[0x2011a3f]/usr/local/lib/libwx_gtk2u_stc-3.0.so.0(+0x98d24)[0x2026d24]/usr/local/lib/libwx_gtk2u_stc-3.0.so.0(+0x3f03e)[0x1fcd03e]/usr/local/lib/libwx_gtk2u_stc-3.0.so.0(_ZNK16wxStyledTextCtrl7SendMsgEiml+0x2f)[0x1fae5ff]/usr/local/lib/libwx_gtk2u_stc-3.0.so.0(_ZN16wxStyledTextCtrl17SetSelectionStartEi+0x34)[0x1fb0f34]/usr/local/lib/libwx_gtk2u_stc-3.0.so.0(+0x33dd4)[0x1fc1dd4]/usr/lib/python2.6/site-packages/wx-3.0-gtk2/wx/_core_.so(+0x10a489)[0x10b0489]/usr/lib/libpython2.6.so.1.0(PyCFunction_Call+0x148)[0x2d77218]/usr/lib/libpython2.6.so.1.0(PyEval_EvalFrameEx+0x499f)[0x2dd475f]/usr/lib/libpython2.6.so.1.0(PyEval_EvalCodeEx+0x85d)[0x2dd63fd]/usr/lib/libpython2.6.so.1.0(PyEval_EvalFrameEx+0x48de)[0x2dd469e]/usr/lib/libpython2.6.so.1.0(PyEval_EvalCodeEx+0x85d)[0x2dd63fd]/usr/lib/libpython2.6.so.1.0(PyEval_EvalFrameEx+0x48de)[0x2dd469e]/usr/lib/libpython2.6.so.1.0(PyEval_EvalCodeEx+0x85d)[0x2dd63fd]/usr/lib/libpython2.6.so.1.0[0x2d63397]***
End of error message
***--------------------------------------------------------------------------mpirun
noticed that process rank 0 with PID 15605 on node madhava exited on signal
6
(Aborted).--------------------------------------------------------------------------
So kindly suggest some details about the error and suggest specific hints
to resolve the problem.Thanks in advance.
Aritra BejDoctoral FellowCSIR - IICB
</pre></blockquote></blockquote><pre style="margin: 0em;">

Hi Aritra,

Did you get a change to look at Troels' suggestions?  Do you see this
error when you run the test suite?  To me this looks like a faulty
installation of wxPython.  These errors and the files listed have
nothing to do with relax.  What is your output when you run:

$ relax -i

Can you run the wxPython demo or the wxPython tests to make sure this
is set up correctly?

Regards,

Edward

   
</pre>
<!--X-Body-of-Message-End-->
<!--X-MsgBody-End-->
<!--X-Follow-Ups-->
</div><!-- end msgdata -->
<br />
<h3><a name="related" href="#related">Related Messages</a></h3>
<div class="relateddata">
<!--X-Follow-Ups-End-->
<!--X-References-->
<ul><li><strong>References</strong>:
<ul>
<li><strong><a name="00004" href="msg00004.html">Re: Issue in ModelFree Calculation</a></strong>
<ul><li><em>From:</em> Edward d'Auvergne</li></ul></li>
</ul></li></ul>
<!--X-References-End-->
<!--X-BotPNI-->
</div><!-- end relateddata -->
<!-- NoBotLinksApartFromRelatedMessages -->

<!--X-BotPNI-End-->
<!--X-User-Footer-->
<!--X-User-Footer-End-->
<div class="footer"></div><br />
<div class="right">Powered by <a href="http://www.mhonarc.org">MHonArc</a>, Updated Tue Aug 30 03:00:12 2016</div>  
</body>
</html>
